src/autopas/selectors/AutoTuner.h

```diff
@@ -200,22 +213,45 @@ class AutoTuner {
   std::unique_ptr<TuningStrategyInterface> _tuningStrategy;
 
   /**
-   * _iteration - Counter of the iterations.
+   * Counter for the simulation iteration.
+   */
+  size_t _iteration;
+
+  /**
+   * Counter for the number of times a tuning phase was started.
+   */
+  size_t _tuningInterval;
+
+  /**
+   * Number of iterations since the end of the last tuning phase.
+   */
+  size_t _iterationsSinceTuning;
+
+  /**
+   * Object holding the actual particle container and having the ability to change it.
    */
-  unsigned int _tuningInterval, _iterationsSinceTuning, _iteration;
   ContainerSelector<Particle> _containerSelector;
+
   double _verletSkin;
   unsigned int _verletClusterSize;
 
   /**
    * How many times each configuration should be tested.
    */
   const size_t _maxSamples;
+
   /**
-   * How many times this configurations has already been tested.
-   * Initialize with max value to start tuning at start of simulation.
+   * Raw time samples of the current configuration from which one evidence will be produced.
+   *
+   * @note Initialized with size of _maxSamples to start tuning at start of simulation.
    */
   std::vector<size_t> _samples;
+
+  /**
+   * For each configuration the collection of all evidence (smoothed values) collected so far and in which iteration.
+   * Configuration -> vector< iteration, time >
+   */
+  std::map<Configuration, std::vector<std::pair<size_t, size_t>>> _evidences;
```

Comment:
are these evidences cleared or is their size reduced at any point? Currently, it looks like an ever growing vector...

Reply:
True, it is an ever-growing vector. For the smoothing, we could restrict it to the number of points that we use for the fitting. However, we might want to move or expose this vector also to the tuning strategies to avoid redundancy, since some already collect this information too.